manifest.json

```
@@ -1,7 +1,7 @@
 {
   "vendor": "vtex",
   "name": "product-quantity",
-  "version": "1.6.0",
+  "version": "1.7.0",
```

No need to change this, the CI/CD bot will handle the version bump once the PR is merged :)

```suggestion
  "version": "1.6.0",
```